### DISCUSSION TITLE
AngleOfSideslip definition

### DISCUSSION THREAD

**Discussion by aidanjungo** [DLR-SL/CPACS#799] [Q&A] [answered]
In the CPACS Documentation I cannot find the definition of the AngleOf sideslip (I mean in which direction it's positive). I imagine it is positive on figure bellow. 

![cpacs_aos](https://user-images.githubusercontent.com/53896963/201963581-820f1c7e-fc7b-444f-b52a-a473dc51b479.png)

As CPACS do not follow the conventional way of defining the aerodynamic angle, I think it could be better to have a similar schema on the main page of the documentation (if it not exist somewhere else).

**Comment by MarAlder** [comment]
Is it not covered by this picture from the main page of the documentation?:

![grafik](https://user-images.githubusercontent.com/43143741/201972172-03e9f84f-8b84-4dbe-96fb-0f7d101d4663.png)

As it is a right-handed CS it should be the one you described in your figure, I would say. Should we add this to the description?

**Reply by aidanjungo** [reply]
Thanks, yes I think we can guess from this figure, but as these angles can be tricky, I think it could avoid some doubts to have top view figure.

**Comment by CLiersch** [comment, marked as answer]
The signs of AngleOfAttack (AoA) and AngleOfSideslip (AoS) are both defined in line with the German LN9300 standard - even though the CPACS coordinate system itself is different (x from nose to tail, z upwards). In CPACS, the angles are defined as positive going **_from inflow to body_** axis with a positive rotation around the CPACS axes. That means:

- AoA is positive if the flow comes from below (y axis positive to the right (view from top) -> positive rotation from flow to aircraft is upwards)
- AoS is positive if the flow comes from the right (viewed from top) (z axis positive upwards -> positive rotation from flow to aircraft is from right to left).

**Reply by MarAlder** [reply]
Thanks for the clearification. Let's put this into the docs via #800 (when #781 is resolved).
